### PR TITLE
Added edit decoders without save test

### PR DESCRIPTION
**Pull request by Mayons95**
Added edit decoders without saving test

### CODE REVIEW

File: cypress/integration/features/management/decoders/edit-decoders-without-saving.feature [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+@rules
+  Scenario: Validate confirmation when closing modal without saving - Management - edit decoders
+    Given The kibana admin user is logged in using xpack and the wazuh logo is displayed
+    When The user navigates to decoders
+    And The user clicks the custom decoders button
+    And The user selects a custom decoders to edit
+    And The user modify the selected decoders
+    And The user tries to exit create decoders page without saving data
+    Then The informative modal is displayed
```
Comment (mauceballos) [inline cypress/integration/features/management/decoders/edit-decoders-without-saving.feature:15]: add new line at en d the line

File: cypress/integration/step-definitions/management/decoders/the-confirm-modal-is-displayed-then.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { Then } from 'cypress-cucumber-preprocessor/steps';
+import { clickElement, elementIsVisible, getSelector } from '../../../utils/driver';
+import { DECODERS_PAGE as pageName} from '../../../utils/pages-constants';
+const confirmModalSelector = getSelector('confirmModalSelector', pageName);
+
+Then('The informative modal is displayed', () => {
+  elementIsVisible(confirmModalSelector);
+  clickElement(confirmModalSelector);
+});
```
Comment (mauceballos) [inline cypress/integration/step-definitions/management/decoders/the-confirm-modal-is-displayed-then.js:9]: add new line at en d the line

File: cypress/integration/step-definitions/management/decoders/the-user-edit-the-selected-decoder-when.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { When } from 'cypress-cucumber-preprocessor/steps';
+import { clickElement, elementIsVisible, getSelector, fillField } from '../../../utils/driver';
+import { DECODERS_PAGE as pageName} from '../../../utils/pages-constants';
+const codeEditorSelector = getSelector('codeEditorSelector', pageName);
+
+When('The user modify the selected decoders', () => {
+  elementIsVisible(codeEditorSelector);
+  clickElement(codeEditorSelector);
+  fillField(codeEditorSelector,"Test");
+});
```
Comment (mauceballos) [inline cypress/integration/step-definitions/management/decoders/the-user-edit-the-selected-decoder-when.js:10]: add new line at en d the line

File: cypress/integration/step-definitions/management/decoders/the-user-exit-without-saving-decoder-when.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import { When } from 'cypress-cucumber-preprocessor/steps';
+import { clickElement, elementIsVisible, getSelector } from '../../../utils/driver';
+import { DECODERS_PAGE as pageName} from '../../../utils/pages-constants';
+const backButtonSelector = getSelector('backButtonSelector', pageName);
+
+When('The user tries to exit create decoders page without saving data', () => {
+  elementIsVisible(backButtonSelector);
+  clickElement(backButtonSelector);
+});
```
Comment (mauceballos) [inline cypress/integration/step-definitions/management/decoders/the-user-exit-without-saving-decoder-when.js:9]: add new line at en d the line

File: cypress/integration/step-definitions/management/decoders/the-user-selects-a-custom-decoder-when.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+import { DECODERS_PAGE as pageName} from '../../../utils/pages-constants';
+const firstCustomDecoder = getSelector('firstCustomDecoder', pageName);
+const xmlDecoderFile = getSelector('xmlDecoderFile', pageName);
+
+When('The user selects a custom decoders to edit', () => {
+  elementIsVisible(firstCustomDecoder);
+  clickElement(firstCustomDecoder);
+  elementIsVisible(xmlDecoderFile);
+  clickElement(xmlDecoderFile);
+});
```
Comment (mauceballos) [inline cypress/integration/step-definitions/management/decoders/the-user-selects-a-custom-decoder-when.js:12]: add new line at en d the line

**Review by mauceballos — CHANGES_REQUESTED** [review]

File: cypress/integration/features/management/decoders/edit-decoders-without-saving.feature (hunk 1 above)
Comment (Mayons95) [reply cypress/integration/features/management/decoders/edit-decoders-without-saving.feature:15]: Done

File: cypress/integration/step-definitions/management/decoders/the-confirm-modal-is-displayed-then.js (hunk 2 above)
Comment (Mayons95) [reply cypress/integration/step-definitions/management/decoders/the-confirm-modal-is-displayed-then.js:9]: Done

File: cypress/integration/step-definitions/management/decoders/the-user-edit-the-selected-decoder-when.js (hunk 3 above)
Comment (Mayons95) [reply cypress/integration/step-definitions/management/decoders/the-user-edit-the-selected-decoder-when.js:10]: Done

File: cypress/integration/step-definitions/management/decoders/the-user-exit-without-saving-decoder-when.js (hunk 4 above)
Comment (Mayons95) [reply cypress/integration/step-definitions/management/decoders/the-user-exit-without-saving-decoder-when.js:9]: Done

File: cypress/integration/step-definitions/management/decoders/the-user-selects-a-custom-decoder-when.js (hunk 5 above)
Comment (Mayons95) [reply cypress/integration/step-definitions/management/decoders/the-user-selects-a-custom-decoder-when.js:12]: Done

**Review by mauceballos — APPROVED** [review]

**Pull request merged**